org-formation/600-access/_tasks.yaml

```diff
@@ -74,6 +74,20 @@ JCWorkflowsNextflowDevAdmin:
     Account: !Ref WorkflowsNextflowDevAccount
     Region: us-east-1
 
+WorkflowsNextflowCIServiceAccounts:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/master/templates/IAM/service-account.yaml
+  StackName: workflows-nextflow-ci-service-accounts
```

Comment:
`accounts`?  minor nitpick but this only creates one service account in each AWS account

Reply:
Good catch! I forgot that this name was going to appear in each account separately.